components/clp-package-utils/clp_package_utils/__init__.py

```diff
@@ -0,0 +1,12 @@
+import logging
+
+# Set up console logging
+logging_console_handler = logging.StreamHandler()
+logging_formatter = logging.Formatter(
+    "%(asctime)s.%(msecs)03d %(levelname)s [%(module)s] %(message)s", datefmt="%Y-%m-%dT%H:%M:%S"
+)
+logging_console_handler.setFormatter(logging_formatter)
+# Set up root logger
```

Comment:
```suggestion

# Set up root logger
```

* As it was in my suggestion.
* To be consistent with our practice of including a newline before comments.